cmake/CodeCoverage.cmake

```diff
@@ -9,6 +9,7 @@
 # Modifications:
 # 1) Change "Quinoa" to "SpECTRE" and "quinoa" to "spectre"
 # 2) Split lines to make commands more legible
+# 3) Silence CMake warnings on uninitialized variables in line 61
```

Comment:
This seems like a very fragile way to identify a change.  Perhaps put an inline comment below and refer to it here?

Reply:
updated this to refer to `cmake_parse_arguments` directly